### ISSUE TITLE
Notify assignee when reminder time is reached

### ISSUE THREAD

**Issue by codeliner** [prooph/proophessor-do#130]
Details will be added as soon as someone wants to take over the exercise

**Comment by yowmamasita** [comment]
I want to try doing this

**Comment by prolic** [comment]
Go for it!

On May 10, 2017 13:12, "Ben Adrian Sarmiento" <notifications@github.com>
wrote:

I want to try doing this

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/prooph/proophessor-do/issues/130#issuecomment-300375700>,
or mute the thread

**Comment by prolic** [comment]
Go for it!

**Comment by shulard** [comment]
Hello guys,

I'm interested to give a try to that exercice. Is there additional instructions somewhere ?

**Comment by codeliner** [comment]
@shulard sorry, I need to check the current implementation first. We have the script [send_reminder_notification.php](https://github.com/prooph/proophessor-do/blob/master/scripts/send_reminder_notification.php). Looks like we did not close the issue.
Let me check that at the weekend.

**Comment by shulard** [comment]
Oh ok 👍 
No problem, I'll take the next one. The idea to learn by doing on that kind of stuff is really awesome 😄. Let me know !

**Comment by codeliner** [comment]
@shulard I checked the history. Indeed, the feature was already added: https://github.com/prooph/proophessor-do/commit/a0208f7a2906d6ab8ed53d797ec69dfe241adec6

It was bonus task of the original reminder feature. 

I cleaned up the list of open tasks. At the moment only [github login](https://github.com/prooph/proophessor-do/issues/129) is left.

Let me know if you want to take this one (complex) or try another one. I'll add new tasks the next days. But need to collect some new ideas first ;)
